test_suite/cloud/test_aws.py

```diff
@@ -397,6 +397,54 @@ def test_yum_plugins(self, host):
             assert host.file('/etc/yum/pluginconf.d/subscription-manager.conf').contains(expect_config), \
                 'Unexpected yum "subscription-manager" plugin status'
 
+    @pytest.mark.run_on(['rhel'])
+    def test_dracut_conf_sgdisk(self, host):
+        """
+        Enable resizing on copied AMIs, added 'install_items+=" sgdisk "' to "/etc/dracut.conf.d/sgdisk.conf"
+        """
+        assert host.package('gdisk').is_installed, 'Package "gdisk" is expected to be installed'
+
+        if float(host.system_info.release) < 8.5:
+            file_to_check = '/etc/dracut.conf.d/sgdisk.conf'
+        else:
+            file_to_check = '/usr/lib/dracut/dracut.conf.d/sgdisk.conf'
+
+        with host.sudo():
+            assert host.file(file_to_check).contains('install_items+=" sgdisk "'), \
+                f'Expected configuration was not found in "{file_to_check}"'
+
+    @pytest.mark.run_on(['rhel8.5', 'rhel8.6', 'rhel9.0'])
+    def test_dracut_conf_xen(self, host):
+        """
+        BugZilla 1849082
+        Add ' xen-netfront xen-blkfront ' to '/etc/dracut.conf.d/xen.conf' in x86 AMIs prior RHEL-8.4.
+        Using image builder from RHEL-8.5, add ' nvme xen-blkfront ' to '/usr/lib/dracut/dracut.conf.d/ec2.conf'.
+        This is not required in arm AMIs.
+        """
```

Comment:
I would not remove the ticket reference `#COMPOSER-1096` from the test. I think every context we can provide has value. Maybe we can include it on the docstring

Reply:
Good point! I forgot to add it here.